src/patterns/scheduled-lambda.test.ts

```diff
@@ -15,7 +15,7 @@ describe("The GuScheduledLambda pattern", () => {
       functionName: "my-lambda-function",
       handler: "my-lambda/handler",
       runtime: Runtime.NODEJS_12_X,
-      rules: [{ schedule: Schedule.rate(Duration.minutes(1)) }],
+      rules: [{ schedule: Schedule.rate(Duration.minutes(1)), input: RuleTargetInput.fromText("Testing") }],
```

Comment:
Would this be better in it's own test given that this test is `should create the correct resources with minimal config`?